doc/api/globals.md

```diff
@@ -81,6 +81,18 @@ This variable may appear to be global but is not. See [`exports`].
 ## global
 <!-- YAML
 added: v0.1.27
+deprecated: REPLACEME
+-->
+
+<!-- type=global -->
+
+> Stability: 0 - Deprecated: Use [`globalThis)`][] instead.
```

Comment:
Has the project decided to deprecate `global` now that `globalThis` is available?

I think there are still cases `global` is useful - in particular I've seen it used when probing for an environment that is Node.js and not "universal JavaScript".

Reply:
I personally have never seen code that checks for the presence of `global`, it's always `if (typeof process === 'object')`, but doc-deprecating it won't break that.

The motivation for deprecation is that there should be One Way To Do It. `globalThis` seems strictly superior since it works in node and modern browsers.

Reply:
Definitely don’t deprecate global; it’s so heavily used that it isn’t web compatible to name globalThis global :-)

In other words, there are many millions of examples in GitHub code search along of code that does this.

Reply:
I agree it’s not possible to runtime deprecate global at this point but this document only doc-deprecate it? We may come back revisiting it or it could just stayed doc deprecate forever but this sends a message to developers that they should use globalThis instead in new code.

Reply:
I don’t think the message is needed. If they want to write universal code, they either can already rely on “every single bundler” handling global transparently for them, or they can use globalThis that works in both.

I just don’t think there’s any value or need to push people in a direction here.